data/ecosystems/p/polkadot.toml

```diff
@@ -5324,6 +5326,13 @@ url = "https://github.com/Polkadex-Platform/Polkadex"
 [[repo]]
 url = "https://github.com/Polkadex-Substrate/Polkadex"
 
+[[repo]]
+url = "https://github.com/Polkadot-Blockchain-Academy/pba-book"
+tags = ["Documentation"]
+
+[[repo]]
+url = "https://github.com/Polkadot-Blockchain-Academy/pba-qualifier-exam"
+
```

Comment:
you can remove these lines since the taxonomy operates like a graph and will upstream the subecosystem's repos/devs